docs/developer/architecture/user_customisations.rst

```diff
@@ -0,0 +1,51 @@
+User Customisations and Persistence
+===================================
+
+Context
+-------
+
+MetalK8s comes with a set of tools and services that may need to be configured
+on site. At the same time, we don't want the administrator of the cluster to
+master each and every service of the cluster. We also don't want to allow any
```

Comment:
We don't want to allow **all** kinds of customisations (_not any_ means nothing)

Reply:
done